.golangci.yml

```diff
@@ -38,10 +38,15 @@ output:
 
 linters-settings:
   depguard:
-    list-type: blacklist
-    include-go-root: true
-    packages-with-error-message:
-      - sync/atomic: "Use go.uber.org/atomic instead of sync/atomic"
-      - io/ioutil: "Use corresponding 'os' or 'io' functions instead."
+    rules:
+      main:
+        deny:
+          - pkg: "sync/atomic"
+            desc: "Use go.uber.org/atomic instead of sync/atomic."
+          - pkg: "io/ioutil"
+            desc: "Use corresponding 'os' or 'io' functions instead."
+  gocritic:
+    disabled-checks:
+      - appendAssign
```

Comment:
Since I didn't think the following error was particularly harmful, I ignored "appendAssign".

> pkg/app/piped/controller/controller.go:556:13: appendAssign: append result not assigned to the same slice (gocritic)
	targets := append(runnings, planneds...)